runtime/doc/news.txt

```diff
@@ -377,6 +377,7 @@ The following changes to existing APIs or features add new behavior.
   • Enabled treesitter highlighting for treesitter query files.
   • Enabled treesitter highlighting for help files.
   • Enabled treesitter highlighting for Lua files.
+  • 'grepprg' defaults to using ripgrep if available.
```

Comment:
Nit: in _this_ case I would move this line to after the `'shortmess'` entry (to keep treesitter stuff at the end).

Reply:
done